examples/trials/feature-selection/gbdt_selector/gbdt_selector_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation
```

Comment:
how about moving these examples to `examples/feature-selection/gbdt....`?

Reply:
OK